### PR TITLE
Add TestRegionalXLBILBTransition e2e test

### PR DESCRIPTION
**Pull request by panslava**
- This test verifies transition between default backend, RXLB, ILB

This test is almost a full copy of TestILBUpdate

/assign @spencerhance

### CODE REVIEW

File: cmd/e2e-test/regional_xlb_test.go [hunk 1]
```diff
@@ -126,3 +127,171 @@ func TestRegionalXLB(t *testing.T) {
...
+			}
+
+			// Verify everything works
+			ing, err = e2e.WaitForIngress(s, tc.ingUpdate, nil, nil)
+			if err != nil {
+				t.Fatalf("error waiting for Ingress to stabilize: %v", err)
+			}
+			t.Logf("GCLB resources created (%s/%s)", s.Namespace, tc.ingUpdate.Name)
+
+			// Perform whitebox testing.
```
Comment (spencerhance) [inline cmd/e2e-test/regional_xlb_test.go:278]: Can we also verify the GC of the other LB here?

Comment (panslava) [reply cmd/e2e-test/regional_xlb_test.go:349]: Added verification for GC of both LBs

Comment (spencerhance) [reply cmd/e2e-test/regional_xlb_test.go:349]: Thanks!

**Pull request merged**